.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+files: 'xvec\/'
+repos:
+  - repo: https://github.com/python/black
+    rev: 22.10.0
+    hooks:
+      - id: black
+        language_version: python3
+  - repo: https://github.com/pycqa/flake8
+    rev: 5.0.4
+    hooks:
+      - id: flake8
+        language: python_venv
```

Comment:
I would probably also add [`isort`](https://github.com/pycqa/isort), but up to you, of course